app/assets/javascripts/modules/Filter.js

```diff
@@ -0,0 +1,10 @@
+define(['jquery', 'List', 'ListFuzzySearch'], function($, List, ListFuzzySearch) {
+  'use strict';
+
+  var firmOptions = {
+    valueNames: ['js-frn', 'js-firm-name', 'js-principal-name'],
+    plugins: [ListFuzzySearch()]
+  };
+
+  var firmList = new List('firm-list', firmOptions);
```

Comment:
I've got it running locally in Firefox, Chrome and Safari after removing the var \o/